doc/examples/plot_matching.py

```diff
@@ -64,44 +84,50 @@
                                      window_size=9)
 
 
-def gaussian_weights(window_ext, sigma=1):
-    y, x = np.mgrid[-window_ext:window_ext+1, -window_ext:window_ext+1]
+def gaussian_weights(patch_radius, sigma=1):
+    y, x = np.mgrid[-patch_radius:patch_radius+1, -patch_radius:patch_radius+1]
     g = np.zeros(y.shape, dtype=np.double)
     g[:] = np.exp(-0.5 * (x**2 / sigma**2 + y**2 / sigma**2))
     g /= 2 * np.pi * sigma * sigma
     return g
 
 
-def match_corner(coord, window_ext=5):
-    r, c = np.round(coord).astype(np.intp)
-    window_orig = img_orig[r-window_ext:r+window_ext+1,
-                           c-window_ext:c+window_ext+1, :]
 
-    # weight pixels depending on distance to center pixel
-    weights = gaussian_weights(window_ext, 3)
-    weights = np.dstack((weights, weights, weights))
+def match_corner(coord_list1,coord_list2,img1,img2, patch_radius=5):#     # find correspondences using simple weighted sum of squared differences
```

Comment:
cleanup explaining what's going on and making the pieces of the code here easier to split apart